cabal-helper.cabal

```diff
@@ -113,9 +113,9 @@ common build-deps
     ghc-options:       -Wall
 
 
-library c-h-internal
+common c-h-internal
   import:             build-deps, extensions
-  exposed-modules:
+  other-modules:
```

Comment:
Otherwise we would expose these modules everywhere, did I get this right?

Reply:
Yeah that's right.

Reply:
The `exposed-modules: Symlink` below should also be other-modules.